lib/get_last_successful_artifact.rb

```diff
@@ -1,4 +1,30 @@
+require "uri"
 
-module GetLastSuccessfulArtifact
-  # Your code goes here...
+require "jenkins_api_client"
+
+class GetLastSuccessfulArtifact
+  def initialize(jenkins_origin: nil, job_name: nil)
+    @job_name = job_name.encode(Encoding::UTF_8)
+    @client = JenkinsApi::Client.new(server_url: jenkins_origin)
+  end
+
+  def get
```

Comment:
This method seems return the first artifact in some kind of(alphabetical?) order.
I think `get_with_path` is safer `get` and there is no reason for using `get`.

Suggestions:
- Delete `get` method and support `get_with_path` method only.
- Change `get` method to raise an error when there are more than one artifacts.